packages/nimble-components/src/table/template.ts

```diff
@@ -97,6 +99,7 @@ export const template = html<Table>`
                                             class="header"
                                             sort-direction="${x => (typeof x.columnInternals.currentSortIndex === 'number' ? x.columnInternals.currentSortDirection : TableColumnSortDirection.none)}"
                                             ?first-sorted-column="${(x, c) => x === c.parent.firstSortedColumn}"
+                                            ?indicators-hidden="${x => x.columnInternals.minPixelWidth < defaultMinPixelWidth}"
```

Comment:
I'm iffy about this change. There is now some magical value table column users are unaware of that cause the indicators to disappear. Lets say I want my table columns to be flexible, the fractional width will be large but I set the minPixelWidth small to let the table user shrink the width as much as they want. So normally the column is large but the user interactively resizes smaller. Now unclearly at some point I configure all the status icons disappear.

Reply:
Updated the logic so that a column explicitly tells the header (though `columnInternals`) that it wants the sort/group indicators to be hidden.